openBCISample.js

```diff
@@ -31,30 +31,39 @@ var sampleModule = {
   *                  calling OpenBCIConstans.channelSettingsArrayInit(). The most important rule here is that it is
   *                  Array of objects that have key-value pair {gain:NUMBER}
   * @param convertAuxToAccel (optional) {Boolean} - Do you want to convert to g's? (Defaults to true)
-  * @returns {Promise} - A sample object
   */
   parseRawPacketStandard: (dataBuf, channelSettingsArray, convertAuxToAccel) => {
     const defaultChannelSettingsArray = k.channelSettingsArrayInit(k.OBCINumberOfChannelsDefault);
+    if (k.isUndefined(convertAuxToAccel) && k.isUndefined(channelSettingsArray)) {
```

Comment:
The rearrangement of the logic here will only provide a default value for `channelSettingsArray` if `convertAuxToAccel` is also unspecified, leaving `channelSettingsArray` undefined if `convertAuxToAccel` is provided.  I mention it because this was not the behavior of the code previously.  Is this an intended change?